### PR TITLE
[10.0] [ADD] analytic_resource_plan_stock_picking

### PR DESCRIPTION
**Pull request by nikul-serpentcs**
Analytic Resource Planning - get resources
==========================================
Module features:
    - prepare the stock for the project
    - create Purchase request if there are not enough qty in stock

### CODE REVIEW

**Comment by nikul-serpentcs** [comment]
@aheficent Could you please review.

File: analytic_resource_plan_stock_picking/README.rst [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    - create Purchase request if there are not enough qty in stock
+
+
+Usage
+=====
+
+Create the resource plan lines and approve:
+
+* The system will fetch the products from stock is available
+* Purchase Request will be created when there's not enough qty in stock
```
Comment (AaronHForgeFlow) [inline analytic_resource_plan_stock_picking/README.rst:19]: Can you create a contributors section and add 
* Jordi Ballester <jordi.ballester@eficent.com>
* Aaron Henriquez <ahenriquez@eficent.com>

and serpent?

File: analytic_resource_plan_stock_picking/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# Copyright 2017 Serpent Consulting Services Pvt. Ltd.
```
Comment (AaronHForgeFlow) [inline analytic_resource_plan_stock_picking/__manifest__.py]: Reserverd for authors

**Comment by nikul-serpentcs** [comment]
@aheficent Could you please look it.

File: analytic_resource_plan_stock_picking/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Analytic Resource Planning - get resources",
+    "version": "10.0.1.0.0",
+    "author": "Eficent",
+    "license": 'AGPL-3',
+    "website": "www.eficent.com",
+    "description": """Fetch stock for projects""",
```
Comment (AaronHForgeFlow) [inline analytic_resource_plan_stock_picking/__manifest__.py:11]: summary instead of description

**Comment by AaronHForgeFlow** [comment]
I closed this because the module was proposed in v7 here: https://github.com/Eficent/eficent-odoo-addons/tree/7.0-stock_location_analytic/analytic_resource_plan_stock_picking and migrated to v9 here: https://github.com/Eficent/eficent-odoo-addons/tree/9.0/analytic_resource_plan_stock_picking

Can you create the PR here https://github.com/Eficent/eficent-odoo-addons? Sorry for the inconvenience.

File: analytic_resource_plan_stock_picking/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    "category": "Generic Modules/Projects & Services",
+    "depends": [
+        "analytic_resource_plan",
+        "analytic_location",
+        "purchase_request",
+        "project_location"
```
Comment (AaronHForgeFlow) [inline analytic_resource_plan_stock_picking/__manifest__.py:17]: @nikul-serpentcs Please set the dependency to analytic_resource_plan_purchase_request back. The module is not ported yet but this dependency is very important.

**Comment by nikul-serpentcs** [comment]
@aheficent moved to [here]( https://github.com/Eficent/eficent-odoo-addons/pull/65
)

**Pull request closed without merging**